tools/image_processing/imagej2/imagej2_noise/imagej2_noise_macros.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<macros>
+    <xml name="insertion_select">
```

Comment:
You can also define macros inside of your tool.xml. This comes very handy in such situations.